crates/ibc-core/ics02-client/types/src/msgs/create_client.rs

```diff
@@ -1,15 +1,14 @@
 //! Definition of domain type message `MsgCreateClient`.
 
+use ibc_primitives::prelude::*;
+use ibc_primitives::{Msg, Signer};
 use ibc_proto::google::protobuf::Any;
 use ibc_proto::ibc::core::client::v1::MsgCreateClient as RawMsgCreateClient;
 use ibc_proto::Protobuf;
 
-use crate::core::ics02_client::error::ClientError;
-use crate::core::Msg;
-use crate::prelude::*;
-use crate::signer::Signer;
+use crate::error::ClientError;
 
-pub(crate) const TYPE_URL: &str = "/ibc.core.client.v1.MsgCreateClient";
+pub const CREATE_CLIENT_TYPE_URL: &str = "/ibc.core.client.v1.MsgCreateClient";
```

Comment:
Did so for two reasons. Firstly, there were discrepancies in the way we named TYPE_URLs across various sections of our codebase. (e.g [here](https://github.com/cosmos/ibc-rs/blob/49a5ffa8fcefa8515d55ecc3b2ed57364eb9c8df/crates/ibc-clients/ics07-tendermint/types/src/header.rs#L26)). secondly, I aimed to ensure consistency in our import paths.
Also, given that these consts are public now, it will be less cumbersome for importers. They won't need to cast to a different naming every time they make an import.

Reply:
Ok. Sounds fair to me :+1: